### ISSUE TITLE
MD044: False positive

### ISSUE THREAD

**Issue by Ravencentric** [rvben/rumdl#567]
### rumdl version

0.1.69

### Installation method

Other

### Bug description

`ravencentric.github.io` (NOT a URL) gets changed to `ravencentric.GitHub.io`.

### Steps to reproduce

1. Create a `test.md` with the content:
	```md
	# Foo
	
	My personal website is located at
	<!-- Don't want "GitHub" capitalized here -->
	[ravencentric.github.io](https://ravencentric.github.io)
	<!-- Do want "GitHub" capitalized here -->
	hosted by GitHub Pages
	```
2. Create a `test.toml` config file with the content:
	```toml
	[MD044]
	names = ["GitHub"]
	```
3. Run `rumdl fmt --config test.toml test.md`
	```console
	$ rumdl fmt --config test.toml test.md
	test.md:5:15: [MD044] Proper name 'github' should be 'GitHub' [fixed]

	Fixed: Fixed 1/1 issues in 1 file (50ms)
	```

### Expected behavior

Input remains untouched:
```md
# Foo

My personal website is located at
<!-- Don't want "GitHub" capitalized here -->
[ravencentric.github.io](https://ravencentric.github.io)
<!-- Do want "GitHub" capitalized here -->
hosted by GitHub Pages
```

### Actual behavior

`[ravencentric.github.io]` gets changed to `[ravencentric.GitHub.io]`
```md
# Foo

My personal website is located at
<!-- Don't want "GitHub" capitalized here -->
[ravencentric.GitHub.io](https://ravencentric.github.io)
<!-- Do want "GitHub" capitalized here -->
hosted by GitHub Pages
```

### Configuration

```toml
[MD044]
names = ["GitHub"]
```

### Operating system

Windows

**Comment by rvben** [comment]
Thanks for the report, @Ravencentric! This is fixed in v0.1.70.
